sailfishusermanager.cpp

```diff
@@ -39,6 +39,7 @@ const char *GROUPS_USER = "USER_GROUPS";
 const auto GROUPS_IDS_FILE = QStringLiteral("/usr/share/sailfish-setup/group.ids");
 const auto SKEL_DIR = QStringLiteral("/etc/skel");
 const auto USER_HOME = QStringLiteral("/home/%1");
+const auto GUEST_USER = QStringLiteral("sailfishguest");
```

Comment:
This has a problem. It can collide with user selected user name (if they create user named Sailfish Guest for example). Can it be named sailfish-guest?